### DISCUSSION TITLE
How to add, remove/hide or change the order of wallets from popup window?

### DISCUSSION THREAD

**Discussion by sanjaybip** [family/connectkit#141] [Q&A] [answered]
I am coming from Rainbow Kit and I like the UI much better here. But I don't find any option to customize the wallet lists. How I can show, hide or change the order of wallet in popup window? This is much easy in Rainbow Kit.

**Comment by miguelmota** [comment]
Ability to sort list of wallets displayed is a simple but much needed feature. I can submit a PR if you are open to it @lochie

**Reply by JulienWebDeveloppeur** [reply]
please i need to be able to filter out some wallet provider @lochie

**Reply by lochie** [reply, marked as answer]
sorry, mustve missed this one 😬
check out [wallet configs](https://family.co/docs/connectkit/wallets)
